src/cursor.js

```diff
@@ -368,3 +368,9 @@ function getRangeBoundingClientRect (range, win) {
   el.remove()
   return coords
 }
+
+function isCloseTo (a, b) {
+  if (a === b) return true
+  if (Math.abs(a - b) <= 2) return true
```

Comment:
@marcbachmann, the `2` here is an arbitrary heuristic you made up, yes?

Also, I’m always torn between readable code (yours) and more compact expressions, e.g.
```javascript
function isCloseTo (a, b) {
    return a === b || Math.abs(a - b) <= 2
}
```

Reply:
just wanted to comment on that, but got drawn into a meeting.
Yes, it is. We could increase it to something like 5.
Not sure yet how the safari calculations work and why there's a 2px diff.
We're placing an element in the same div, so it should be really the same.

Reply:
My intuition says to keep the number small, so I think `2` makes sense for now.

Alternatively, we’d make that a config option that defaults to `2` if we wanted more flexibility. But that’s probably a bit over-engineered.

Reply:
regarding the style, I don't see any benefit of the shorter version 😁
correctness is more important than compactness.
and correctness can be reached better with simpler code

Reply:
Agreed on all three points 🤓